src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -3353,17 +2348,7 @@ namespace winrt::Microsoft::Terminal::Control::implementation
 
     Windows::Foundation::IReference<winrt::Windows::UI::Color> TermControl::TabColor() noexcept
     {
-        auto coreColor = _terminal->GetTabColor();
-        return coreColor.has_value() ? Windows::Foundation::IReference<winrt::Windows::UI::Color>(til::color{ coreColor.value() }) : nullptr;
-    }
-
-    // Method Description:
-    // - Sends an event (which will be caught by TerminalPage and forwarded to AppHost after)
-    //   to set the progress indicator on the taskbar
-    winrt::fire_and_forget TermControl::TaskbarProgressChanged()
-    {
-        co_await resume_foreground(Dispatcher(), CoreDispatcherPriority::High);
-        _SetTaskbarProgressHandlers(*this, nullptr);
+        return _core->TabColor();

```

Comment:
this one actually is more of a "uwp control only" setting. I forget why Control even _knows_ about it ...

Reply:
In the hypothetical future we allow for a terminal to set the tab color, like it can set the title. It's not out of the imagination. I feel like I should leave a comment here, cause I feel like this comes up on every TabColor discussion :P